src/hooks/useDarkMagic.ts

```diff
@@ -61,3 +63,21 @@ const cargo = async (arg0: string | undefined) => {
 
   return undefined
 }
+
+const brew = async (arg0: string | undefined) => {
+  if (!Path.findBin("brew")) return undefined
+
+  const res = await useFetch(`https://formulae.brew.sh/api/formula/${arg0}.json`)
+
+  if (res.status == 200) {
+    return {
+      // FIXME: we should package brew
```

Comment:
yes we should pkg brew IMO however if we do so bottles must install to `/usr/local` or `/opt/homebrew` to function. So `tea` will need to `sudo` one time on the first instance of using `brew` to change the permissions on those directories.

So as it stands this patch should either wait for brew to pkg’d or check `brew` is otherwise independently installed. Otherwise this is not fully functional.

Reply:
`  if (!Path.findBin("brew")) return undefined`

Reply:
lol I didn't review thoroughly to say the least

Reply:
I see you also said as much in the PR description 🤦🏻‍♂️